### PR TITLE
Document behavior in case of suspension by owner.

### PR DESCRIPTION
**Pull request by Zimmi48**
### Why:

Document what I learned about GitHub App suspension by asking GitHub support (ticket `#1291684`).

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by TorrZeed — COMMENTED** [review]
Duplicate of #

**Comment by Zimmi48** [comment]
@TorrZeed duplicate of which PR?

File: content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ The integrator who owns and maintains a GitHub app, also called a GitHub App own
 
 People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.
 
+Note that if an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
+
```
Comment (ramyaparimi) [inline content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md:22]:
```suggestion
{% note %}

**Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.

{% endnote %}
```

**Review by ramyaparimi — CHANGES_REQUESTED** [review]
@Zimmi48 I made a suggestion as per the format we use for calling out `note`. For more information see "[Callout tags](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#usage)".

**Comment by ramyaparimi** [comment]
@Zimmi48 
Thanks so much for opening a PR! I got this triaged for a writer's review :zap:

File: content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md [hunk 2]
```diff
@@ -18,8 +18,11 @@ The integrator who owns and maintains a GitHub app, also called a GitHub App own
 
 People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.
 
-Note that if an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
+{% note %}
 
+**Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
+
+{% endnote %}
 {% data reusables.user-settings.access_settings %}
```
Comment (ramyaparimi) [inline content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md:26]:
```suggestion

{% endnote %}

{% data reusables.user-settings.access_settings %}
```

**Review by ramyaparimi — DISMISSED** [review]
@Zimmi48 I saw this at staging and looks like it needs spacing. Could you add these changes?? ✨

**Comment by Zimmi48** [comment]
Looks better now. Thanks @ramyaparimi!

File: content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ The integrator who owns and maintains a GitHub app, also called a GitHub App own
 
 People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.
 
+{% note %}
+
+**Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
+
+{% endnote %}
```
Comment (lecoursen) [inline content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md:25]: This is general information that doesn't meet [our criteria for a callout](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#callouts). 

```suggestion
If an installation has been suspended by the {% data variables.product.prodname_github_app %} owner, installation owners cannot unsuspend their installations of the {% data variables.product.prodname_github_app %}. However, installation owners can change other settings, such as repository selection, while the app is suspended.
```

**Review by lecoursen — COMMENTED** [review]
Thanks for adding this useful information! I'm going to apply a small suggestion and then merge. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**